src/screens/landingPageScreens/ResourcesScreen.tsx

```diff
@@ -0,0 +1,88 @@
+
+
+interface CardData {
+    title: string;
+    description: string;
+    link: string;
+    linkUrl: string;
+}
+
+interface ResourcesScreenProps {
+    title: string;
+    description: string;
+    link: string;
+    linkUrl: string;
+}
+
+const cardsData: CardData[] = [
+    {
```

Comment:
This cound be moved to a dedicated file for static data, e.g., `src/data/CardData`, and renamed to something like `applicationInfoCards: CardData[]`.